.github/workflows/test.yaml

```diff
@@ -41,8 +42,17 @@ jobs:
         run: sudo apt install -y lua5.4 liblua5.4-dev
         if: ${{ matrix.LIBLUA == '5.4' }}
 
-      - name: Running CMake
+      - name: Setup LuaJIT v2.1 packages
+        run: sudo apt install -y libluajit2-5.1-2 libluajit2-5.1-dev luajit2
+        if: ${{ matrix.LIBLUA == 'luajit-v2.1' }}
+
+      - name: Running CMake (PUC Rio Lua)
         run: cmake -DCMAKE_C_COMPILER=clang-15 -DCMAKE_CXX_COMPILER=clang++-15 -DENABLE_TESTING=ON -S . -B build
+        if: ${{ matrix.LIBLUA != 'luajit-v2.1' }}
+
+      - name: Running CMake (LuaJIT)
+        run: cmake -DCMAKE_C_COMPILER=clang-15 -DCMAKE_CXX_COMPILER=clang++-15 -DENABLE_TESTING=ON -DENABLE_LUAJIT=on -S . -B build
```

Comment:
Since the command is almost the same, maybe it is better to introduce an additional variable under the `include` section (see an example [here](https://github.com/tarantool/luajit/blob/tarantool/master/.github/workflows/exotic-builds-testing.yml#L40)). In this case all steps are identical, and all differences are described inside the matrix configuration. Feel free to ignore, though.

Reply:
Right, this time has come, but I'll do this refactoring a bit later. Now I want to close tickets required for lua-c-api-tests. Submitted an issue https://github.com/ligurio/luzer/issues/48.

Reply:
Ok, just a suggestion.